src/db/options.rs

```diff
@@ -83,6 +85,17 @@ pub struct CreateCollectionOptions {
 
     /// The default configuration for indexes created on this collection, including the _id index.
     pub index_option_defaults: Option<IndexOptionDefaults>,
+
+    /// Specifies options for creating a timeseries collection.
```

Comment:
Can we document here that this feature is only available on 5.0+?

Reply:
Good call, done.